pymmcore/__init__.pyi

```diff
@@ -1014,10 +1014,22 @@ class CMMCore:
         """Sets the current slm device."""
     def setSLMExposure(self, slmLabel: str, exposure_ms: float) -> None:
         """For SLM devices with build-in light source (such as projectors),
-
         this will set the exposure time, but not (yet) start the illumination"""
+    @overload
+    def setSLMImage(self, slmLabel: str, pixels: np.ndarray[np.uint8]) -> None:
+        """
+        Write a 8-bit grayscale image to the SLM. Pixels must be a 2D numpy array [h,w] of uint8s.
+        Warning: SLM might convert grayscale to binary internally.
+        """
+    @overload
+    def setSLMImage(self, slmLabel: str, pixels: np.ndarray[np.uint8, np.uint8, np.uint8]) -> None:
+        """
+        Write a color image to the SLM (imgRGB32). The pixels must be 3D numpy array [h,w,c] of uint8s with 3 color channels [R,G,B].
+        The dimensions of the array should match the width and height of the SLM.
```

Comment:
```suggestion

        The dimensions of the array should match the width and height of the SLM.
```

spaces after the first line will determine what gets put in the description column in the [summary table](https://pymmcore-plus.github.io/pymmcore-plus/api/cmmcoreplus/#cmmcoreplus-api-summary)